bootstrap/manifest_template.json

```diff
@@ -269,6 +269,33 @@
     "status": 401
   },
   {
+    "id": "logaggregator",
+    "action": "deploy-app",
+    "app": {
+      "name": "logaggregator",
+      "meta": {"flynn-system-app": "true"}
+    },
+    "artifact": {
+      "type": "docker",
+      "uri": "$image_repository?name=flynn/logaggregator&id=$image_id[logaggregator]"
+    },
+    "release": {
+      "env": {
+        "PORT": "1514"
+      },
+      "processes": {
+        "app": {
+          "host_network": true,
+          "cmd": ["logaggregator", "-listenaddr", ":1514"],
+          "omni": true
```

Comment:
I don't think this needs to be an omni job? Omni jobs run on every host in the cluster.